yarn-project/circuit-types/src/logs/tx_l2_logs.ts

```diff
@@ -1,5 +1,7 @@
 import { BufferReader, prefixBufferWithLength } from '@aztec/foundation/serialize';
 
+import isEqual from 'lodash.isequal';
+
 import { FunctionL2Logs } from './function_l2_logs.js';
 import { LogType } from './log_type.js';
 
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: line 11]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->See earlier notes, move the `L2Block.computeKernelLogsHash` function in here as `hash` instead.<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/AztecProtocol/aztec-packages/4541?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->